### PR TITLE
Update readme to remove the reference to the client ID and secret key that have been removed in a previous commit

### PR DESCRIPTION
**Pull request by vhetet**
In a previous commit the client ID and the secret key have been removed in the code, I am just updating the readme to account for that change

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,20 +16,20 @@ Once installed, clone the repository and install its dependencies running:
...
-In order to run the different examples, open the folder with the name of the flow you want to try out, and run its `app.js` file. For instance, to run the Authorization Code example do:
-
-    $ cd authorization_code
-    $ node app.js
-
-Then, open `http://localhost:8888` in a browser.
-
 ### Using your own credentials
-The examples contains a working client ID and secret key. Note, however, that they might be rate limited if they are used frequently. If you are planning to create an application, we recommend you register your app and get your own credentials instead of using the ones in this project.
+You will need to register your app and get your own credentials from the spotify developer dashboard.
```
Comment (arirawr) [inline README.md]: Change to "Spotify for Developers Dashboard"

File: README.md [hunk 2]
```diff
@@ -16,20 +16,20 @@ Once installed, clone the repository and install its dependencies running:
...
 
-Go to [My Applications on Spotify Developer](https://developer.spotify.com/my-applications) and create your application. For the examples, we registered these Redirect URIs:
+To do so, go to [My Applications on Spotify Developer](https://developer.spotify.com/my-applications) and create your application. For the examples, we registered these Redirect URIs:
```
Comment (arirawr) [inline README.md]: Change to `[your Spotify for Developers Dashboard](https://beta.developer.spotify.com/dashboard)`

**Comment by vhetet** [comment]
Hey @arirawr it's changed.
Can you review it again? Thanks

**Comment by arirawr** [comment]
Hey @vhetet - just one last tweak - the link should go to https://beta.developer.spotify.com/dashboard instead of https://developer.spotify.com/my-applications

**Comment by vhetet** [comment]
@arirawr, my bad I missed that change request. It's fixed.

**Pull request merged**